packages/tokens/src/tokens/font/composite.json

```diff
@@ -0,0 +1,152 @@
+{
+  "vads-font-display": {
+    "name": "vads-font-display",
+    "value": {
+      "fontFamily": "{vads-font-family-serif-regular}",
+      "fontSize": "{vads-font-size-display}",
+      "lineHeight": "{vads-font-line-height-heading-5xl}",
+      "marginBottom": "{vads-font-paragraph-spacing-heading-5xl}",
```

Comment:
Weren't all these that were actively used handled by the spacing tokens? 

Thought that was the motivation for not including paragraph-spacing in the NPM dist; seems weird to reference Figma-only tokens within NPM tokens.

Reply:
Chatted with @jessicawoodin about this over the past few days and decided we'll use spacing tokens instead. Composite tokens have been updated to use spacing tokens as marginBottom instead of paragraph spacing.